commons/addons/fs_buckets.md

```diff
@@ -31,7 +31,7 @@ You will be able to retrieve generated data between two deployments.
 <th>Name</th>
 <th>Disk</th>
 <th>Mounts</th>
-<th>Price /Go</th>
+<th>Price per month</th>
```

Comment:
It's the price / Go / Month, see https://www.clever-cloud.com/pricing#fs-bucket

Reply:
Sorry, it's mistake